apps/vs-code-intellisense/src/extension.ts

```diff
@@ -32,21 +38,12 @@ export function getAttributeName(linePrefix: string) {
   return attributeName;
 }
 
-export function isColor(pasteToken: keyof typeof pasteTokens) {
-  if (
-    pasteToken === 'backgroundColors' ||
-    pasteToken === 'borderColors' ||
-    pasteToken === 'colors' ||
-    pasteToken === 'textColors'
-  ) {
-    return true;
-  }
-  return false;
-}
+export const isColor = (tokenKey: TokenKey) =>
+  tokenKey === 'backgroundColors' || tokenKey === 'borderColors' || tokenKey === 'colors' || tokenKey === 'textColors';
```

Comment:
what do you think to move `isColor` to `utils` folder?

Reply:
Yeah, good call! 👍